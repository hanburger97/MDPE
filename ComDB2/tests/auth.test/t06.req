#Queries from unknown users will pass now
select 'access without password'
set user user1
set password password1
#turn it back on
put authentication on
