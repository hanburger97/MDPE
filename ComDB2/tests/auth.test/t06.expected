('access without password'='access without password')
